setup.py

```diff
@@ -4,30 +4,27 @@
 from setuptools import find_packages, setup
 
 GARAGE_GH_TOKEN = os.environ.get('GARAGE_GH_TOKEN') or 'git'
-GYM_VERSION = '0.15.4'
+GYM_VERSION = '0.17.2'
 
 # Required dependencies
 REQUIRED = [
     # Please keep alphabetized
     'akro',
     'click>=2.0',
-    'cloudpickle<1.5',
+    'cloudpickle==1.3',
```

Comment:
Can we do `>=1.3` instead? This seems too restrictive

Reply:
It seems that tensorflow-probability 0.11 pin cloudpickle to 1.3.

Reply:
Ahh right. It's set to >=1.3 on master, but that's unreleased yet. Can you open an issue to track this?

Reply:
Sure. So currently should we pin cloudpickle to 1.3 or >=1.3?

Reply:
I guesswe can leave it at ==1.3 for now